typescript/infra/config/environments/mainnet3/funding.ts

```diff
@@ -24,4 +24,35 @@ export const keyFunderConfig: KeyFunderConfig = {
     [Contexts.ReleaseCandidate]: [Role.Relayer, Role.Kathy],
   },
   connectionType: RpcConsensusType.Single,
+  // desired balance config
+  desiredBalancePerChain: {
+    arbitrum: '3',
+    avalanche: '5',
+    base: '3',
+    bsc: '5',
+    celo: '3',
+    ethereum: '5',
+    gnosis: '5',
+    moonbeam: '5',
+    optimism: '3',
+    polygon: '20',
+    polygonzkevm: '3',
+    scroll: '3',
+  },
```

Comment:
can you add inevm and viction here? and, if it's possible, use types to make sure that all chains in the environment have an entry here?

Reply:
had a look through some of the existing types and we're casting stuff as ChainMaps all the way down, probs needs more TLC down the line to tighten up the typings